WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewModel.swift

```diff
@@ -27,6 +27,8 @@ final class CardReaderSettingsConnectedViewModel: CardReaderSettingsPresentedVie
     var connectedReaderBatteryLevel: String?
     var connectedReaderSoftwareVersion: String?
 
+    var delayToShowUpdateSuccessMessage: DispatchTimeInterval = .seconds(1)
```

Comment:
How about making this a private variable by injecting this as a parameter in the initializer like:

```swift
init(didChangeShouldShow: ((CardReaderSettingsTriState) -> Void)?, knownReaderProvider: CardReaderSettingsKnownReaderProvider? = nil, delayToShowUpdateSuccessMessage: DispatchTimeInterval = .seconds(1))
```

and then passing the value in unit tests? In the codebase, we generally do dependency injection with initializers for parameters that are different in unit tests